### DISCUSSION TITLE
Management Interface Access

### DISCUSSION THREAD

**Discussion by Massimoi79** [WeblateOrg/weblate#12127] [Q&A]
Should an Administration role show the Management Interface? I have added colleagues under the team named 'managers' with the role 'administration'. However, she cannot see or access the Management Interface. The intention is to allow multiple company colleagues to invite users to our 4 individual translation projects. As I understand the Weblate documentation, the 'administration' role is the only role that allows inviting new users to a project.

**Comment by nijel** [comment]
“Administration” currently allows managing projects, but doesn't provide access to site-wide actions.

You can add the “Use management interface” to the “Administration” role to change that.

**Reply by Massimoi79** [reply]
Thanks for your solution. Could you explain how to add that permission to the "Administration" role?

**Comment by nijel** [comment]
If you want users to be able to invite users to individual projects, the `Administration` role should be enough for that. The can invite to individual teams within a project then.

If you want users to be able to invite users to site-wide teams, then built-in roles won't cover that, and you need to create one that matches your needs (probably with `Use management interface` and `Manage teams` permissions). You can then assign that role to existing or new teams.

If this is not clear from https://docs.weblate.org/en/latest/admin/access.html, suggestions (or pull requests) with improvements are welcome.
